test_main.c

```diff
@@ -17,13 +17,20 @@
 
 #include "tests.h"
 
+int runverbose = 1;
```

Comment:
hacked up way to silence the PASS xxxx messages from tests that pass.  defaults to verbose.